packages/hash/frontend/src/pages/entity-editor.page.tsx

```diff
@@ -62,9 +63,23 @@ const ExampleUsage = ({ ownedById }: { ownedById: string }) => {
     [entityTypeRootedSubgraph, propertyTypeIds],
   );
 
+  const handleCreateEntity = async () => {
+    const createdEntity = await createEntity({
+      data: {
+        entityTypeId:
+          "http://localhost:3000/@example/types/entity-type/dummy/v/1",
+        properties: {},
+      },
+    });
+
+    // eslint-disable-next-line no-console
+    console.log({ createdEntity });
```

Comment:
Is this `console.log` intentional for demo purposes?

Reply:
Yes, it's so there's a way of testing this PRs functionality (see the GitHub description). This demo page should be removed when @yusufkinatas's entity editor is merged.